scalafix-core/src/main/scala/scala/meta/internal/symtab/SymbolTable.scala

```diff
@@ -0,0 +1,10 @@
+package scala.meta.internal.symtab
```

Comment:
copy of https://github.com/scalameta/scalameta/blob/v4.9.9/semanticdb/scalac/library/src/main/scala/scala/meta/internal/symtab/SymbolTable.scala

It's not great as drift between this version and the upstream one may cause runtime errors in reflect and cli, but I think it's worth it to keep core clean until we contractualize with scalameta what could be exposed as a public API.